dokka-subprojects/plugin-base/src/test/kotlin/expectActuals/ExpectActualsTest.kt

```diff
@@ -445,9 +446,15 @@ class ExpectActualsTest : BaseAbstractTest() {
         }
     }
 
-    @Test
-    fun `should merge an implicit-expectActual function with a single property #3685`() = testInline(
-        """
+    /**
+     * This is a corner case because a property and a function have the same DRI in the same source set
+     */
+    @Nested
+    inner class MergingPropertyAndFunction {
+
+        @Test
+        fun `should merge an implicit-expectActual function with a single property #3685`() = testInline(
```

Comment:
The behaviour has been changed, but this case is rare and used to cause an exception.